src/transformers/modeling_tf_utils.py

```diff
@@ -707,7 +707,7 @@ def tf_shard_checkpoint(weights, max_shard_size="10GB"):
     return shards, index
 
 
-def load_tf_sharded_weights(model, shard_files, ignore_mismatched_sizes=False, strict=True):
+def load_tf_sharded_weights(model, shard_files, ignore_mismatched_sizes=False, strict=False, _prefix=None):
```

Comment:
This defaulted to `strict=True` before, which meant that you couldn't e.g. load an MLM head model with a SequenceClassification head. We missed this issue because it's hard to test models that big without starting a cloud GPU - is there some way we can make a very small sharded checkpoint for tests?

Reply:
Just found the `max_shard_size` arg - will make a testing checkpoint with it now!

Reply:
(Or just steal @ArthurZucker's one)